### ISSUE TITLE
Compilation caused by hls_m3u8

### ISSUE THREAD

**Issue by wouterdebie** [wouterdebie/locast2tuner#70]
Compilation fails with:

```
error[E0759]: `input` has an anonymous lifetime `'_` but it needs to satisfy a `'static` lifetime requirement
   --> /Users/wouter.de.bie/.cargo/registry/src/github.com-1ecc6299db9ec823/hls_m3u8-0.4.0/src/media_playlist.rs:730:12
    |
729 |     fn from_str(input: &str) -> Result<Self, Self::Err> {
    |                        ---- this data with an anonymous lifetime `'_`...
730 |         Ok(parse_media_playlist(input, &mut Self::builder())?.into_owned())
    |            ^^^^^^^^^^^^^^^^^^^^^-----^^^^^^^-------------^^^
    |                                 |           |
    |                                 |           ...and is required to live as long as `'static` here
    |                                 ...is captured here...
```

Opened an issue at https://github.com/sile/hls_m3u8/issues/65

**Comment by wouterdebie** [comment]
@Croq360 You might want to follow this

**Comment by wouterdebie** [comment]
This issue should be fixed with 0.1.45.
